sendsmaily.php

```diff
@@ -250,3 +250,10 @@ function smaily_nojs_subscribe_callback() {
 }
 add_action( 'admin_post_nopriv_smly', 'smaily_nojs_subscribe_callback' );
 add_action( 'admin_post_smly', 'smaily_nojs_subscribe_callback' );
+
+function showUpgradeNotification($currentPluginMetadata, $newPluginMetadata){
```

Comment:
Please review code styling to comply with [Wordpress guidelines](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions). And use `smaily_` prefix in function names to avoid any possible conflicts with Wordpress core (or plugin) functions.